examples/simple/root.re

```diff
@@ -1,3 +1,8 @@
 Js.log "Do you see this message? If you do, then things should work!";
 
-ReactDOMRe.render <Page message="Hello!" /> (ReasonJs.Document.getElementById "index");
+open ReasonJs;
+
+switch (document |> Document.getElementById "index") {
```

Comment:
Hummm... this is heavy, but that's the right way of doing it, I guess. Hopefully @bobzhang null-to-optional optimization works here!

Also, raising `Not_found` is right but might not be ideal for newcomers. `Invalid_argument "Index not found"` might help more.